website/docs/integrations/integration-with-sveltekit.mdx

```diff
@@ -12,7 +12,7 @@ You can add GraphQL Yoga with a few lines of code and get the benefits of GraphQ
 
 In a SvelteKit project:
 
-<PackageInstall packages={['-D @graphql-yoga/core@alpha']} />
+<PackageInstall packages={['-D graphql', '-D @graphql-yoga/core@alpha']} />
```

Comment:
why are they dev dependency?
```suggestion
<PackageInstall packages={['graphql', '@graphql-yoga/core@alpha']} />
```

Reply:
I'm so used to svelte (frontend side) where everything is compiled into vanilla js, so you don't need it after a build.
But in SvelteKit, I have to be carefull!

Thx.

It's updated now